lang/it.yml

```diff
@@ -0,0 +1,53 @@
+en:
```

Comment:
Besides my comment on the PR, the only issue that seems obvious to me is that this should be `it` instead of `en`.